orangecontrib/prototypes/widgets/owcontingency.py

```diff
@@ -15,32 +23,45 @@ class OWContingencyTable(widget.OWWidget):
     priority = 2010
 
     inputs = [("Data", Table, "set_data", widget.Default)]
-    outputs = [("Contingency Table", Table, )]
+    outputs = [("Contingency Table", Table, ),
+               ("Selected Data", Table),
+               (ANNOTATED_DATA_SIGNAL_NAME, Table)]
```

Comment:
Use class definitions for Input and Output signals.

Example:
```    
class Inputs:
    data = Input("Data", Table)
```
 You can read more [here](http://orange-development.readthedocs.io/widget.html#input-output-signal-definitions).